### PR TITLE
ci: Only run book job if relevant dirs were modified

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Follow up to #1089

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -1,7 +1,24 @@
 on:
   push:
-    branches: [master]
+    branches:
+      - "master"
+    paths:
```
Comment (bilelmoussaoui) [inline .github/workflows/CI.yml:5]: changes to the ci config should also trigger a build

Comment (Hofer-Julian) [reply .github/workflows/CI.yml:5]: True

Comment (bilelmoussaoui) [reply .github/workflows/CI.yml:5]: I think it should exclude changes to the book job manifest though.

Comment (Hofer-Julian) [reply .github/workflows/CI.yml:5]: I've limited it to the CI.yml now

**Comment by bilelmoussaoui** [comment]
Thanks a lot!

**Pull request merged**